app/src/main/res/values-ca/strings.xml

```diff
@@ -0,0 +1,47 @@
+<!--
+  ~ Copyright 2019 Google LLC
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources>
+    <string name="menu_filter_by_grow_zone">Filtrar per zona de creixement</string>
+    <string name="plant_detail_image_content_description">Imatge de la planta</string>
+    <string name="my_garden_title">El meu jardí</string>
+    <string name="plant_list_title">Llistat de plantes</string>
+    <string name="available">Plantes Disponibles</string>
+    <string name="plant_details_title">Detalls de la planta</string>
+    <string name="add_plant">Afegir planta</string>
```

Comment:
```suggestion
    <string name="add_plant">Afegeix una planta</string>
```

Reply:
We shouldn't add an unnecessary article `una` to translate from the English `Add plant`. Basically, because it's not just any plant, it's the selected one. What do you think?

Reply:
Imho, that's a grammar license used in English software translations. In Catalan, though, we should include them.

In fact, Termcat includes this in [its guidelines](https://www.termcat.cat//Thor/files/repositoriDocumental/20110615143323_LocalitzacioProductesInformatics.pdf).

Reply:
If the command is indeed to add _the selected plant_, the translation should be _Afegeix la planta_.

Reply:
I agree with @ChristianGarcia, and you can also check that the article is used in lots of software when it's not there in English: https://www.softcatala.org/recursos/web_search.py?source=&target=Afegeix+un&project=tots

If the added plant is the selected one, it should be "Afegeix la planta" as Christian pointed out. But in any case, it should be "Afegeix" instead of "Afegir".